indico/modules/events/editing/models/revisions.py

```diff
@@ -176,3 +176,8 @@ def is_submitter_revision(self):
     @property
     def is_editor_revision(self):
         return self.type.is_editor_action
+
+    @property
+    def last_update_dt(self):
+        latest_comment_dt = max((comment.modified_dt or comment.created_dt for comment in self.comments), default=None)
+        return latest_comment_dt or self.modified_dt or self.created_dt
```

Comment:
To avoid the query spam you most likely need to implement this as a `column_property` so you can run this within the SQL query that loads the editables, instead of having to do it on the python side.

Reply:
An alternative would be an `updated_dt` that would be updated like `modified_dt` but also when commenting. But let's try the column property first before adding another real column :)